libs/sitemaps/test/integration/createSitemapGenerator.spec.ts

```diff
@@ -0,0 +1,100 @@
+import { dispose } from '@hicommonwealth/core';
+import { tester } from '@hicommonwealth/model';
+import { expect } from 'chai';
+import {
+  createAsyncWriterMock,
+  createDatabasePaginatorDefault,
+  createSitemapGenerator,
+} from '../../src';
+
+describe('createSitemapGenerator', function () {
+  this.timeout(10000);
+
+  before(async () => {
+    const [user] = await tester.seed('User', {
+      isAdmin: true,
+      selected_community_id: null,
+    });
+
+    if (!user) {
+      throw new Error('No user');
+    }
+
+    const [node] = await tester.seed('ChainNode', { contracts: [] });
+
+    if (!node) {
+      throw new Error('No node');
+    }
+
+    const [community] = await tester.seed('Community', {
+      name: 'Acme',
+      chain_node_id: node.id,
+      discord_config_id: null,
+      Addresses: [],
+      CommunityStakes: [],
```

Comment:
Add `contest_managers: []` and that will fix the foreign key error. Also with the updated tsconfig your test should run without ts-mocha installation so you should be able to update the package.json test command.

Reply:
I added this change but I don't want to fix the test in this PR.

The first issue is that the tests hang on exit. I'm not sure why.

My runner had to do a process.exit(0) so there's some background runnable or open socket preventing node from exiting. 

This PR is asymptotic to being merged and at this rate it will never land. 

The sitemaps code works. I tested it with the dump so I'd just like to get the landed and smaller things fixed in subsequent PRs.